README.md

```diff
@@ -94,12 +99,12 @@ export default Ember.Controller.extend({
   }],
 
   theme: defaultTheme
-
+  
 });
 ```
 
 ```handlebars
-{{high-charts chartOptions=chartOptions content=chartData theme=theme}}
+{{high-charts chartOptions=chartOptions content=chartData theme=theme callback=afterRenderCallback}}
```

Comment:
I think the `callback` argument should be removed from this example. The `afterRenderCallback` doesn't exist in the example controller.